### ISSUE TITLE
Seo: "Speaking" title and h1

### ISSUE THREAD

**Issue by rraue** [mozilla/irlpodcast#83]
The episode pages should have a “speaking” h1-headline with the main topic of the episode. Currently the h1 is part of the player and js-rendered. Google uses for indexing the DOM. 

I suggest, we change the player h1 to a h2 and add a “speaking” h1 on top of the episode description. 

The more speaking headline should be the <title> as well. “Social Bubble Bath” is a good title for the episode, but its not a good headline for search results. A good headline for this example episode 13 would be “How social media can create, and can break, our filter bubbles” like written in the description.

**Comment by jpetto** [comment]
We don't have control over the markup of the player, so we can't change that `<h1>`. The player is rendered in an iframe, so hopefully doesn't have significant SEO impact.

In regards to the "speaking" `<h1>`, can this text be visually hidden (as is the current `<h2>`)? Ideally we wouldn't have two identical titles (one in the player, one in the page).

**Comment by rraue** [comment]
No, we should not hide important text, then rather not having it. Do you see any other possibility to add at least a speaking <title>, as this is the important part of this issue? The title is the headline for users in search and should give them the best idea of what to expect on the search result.

**Comment by jpetto** [comment]
After conversation with @rraue in SF, we can omit the "speaking" `<h1>` entirely for now, and only update the episode titles.
